util/chplenv/third_party_utils.py

```diff
@@ -411,16 +411,16 @@ def read_bundled_pkg_config_file(pkg, ucp='', pcfile=''):
 
     # give up early if the 3rd party package hasn't been built
     if not os.path.exists(install_path):
-        return None
+        return (None, None)
 
     pcpath = os.path.join(install_path, 'lib', 'pkgconfig', pcfile)
 
     # if we get this far, we should have a .pc file. check that it exists.
     if not os.access(pcpath, os.R_OK):
         error("Could not find '{0}'".format(pcpath), ValueError)
-        return None
+        return (None, pcpath)
 
     find_path = os.path.join('third-party', pkg, 'install', ucp)
     replace_path = install_path
 
-    return read_pkg_config_file(pcpath, find_path, replace_path)
+    return (read_pkg_config_file(pcpath, find_path, replace_path), pcpath)
```

Comment:
This looks OK but please update the comment above `read_bundled_pkg_config_file` to indicate what it returns a tuple of (dictionary, path to .pc file).

Reply:
Done.